specs/CommandBar/new-command-bar-features.md

```diff
@@ -0,0 +1,349 @@
+via the `AppBarButton.Flyout` property. See a simple example of this below:
+
+
+```xaml
+<AppBarButton Label="Sort by" Icon="...">
+    <AppBarButton.Flyout>
+        <MenuFlyout>
+            <MenuFlyoutItem>Sort by artist name</MenuFlyoutItem>
+            <MenuFlyoutItem>Sort by album name</MenuFlyoutItem>
+            <MenuFlyoutItem>Sort by genre</MenuFlyoutItem>
+        </MenuFlyout>
+    </AppBarButton.Flyout>
+</AppBarButton>
+```
+
+![Simple MenuFlyout within a CommandBar](images/simple-menuflyout.PNG)
+
+ Items within a `MenuFlyout` (such as "Sort by artist name") can also have their own MenuFlyouts, 
+creating a  cascading or nested MenuFlyout. See an example of this below. 
+ 
+![Image of a cascading menu](images/nested-menuflyout.PNG)
+
+In order to create this cascading/nested menu today, you can use a class called
+[MenuFlyoutSubItem](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.menuflyoutsubitem?view=winrt-19041) as an item in the menu. 
+
+```xaml
+<MenuFlyout>
+    <MenuFlyoutItem>Sort by artist name</MenuFlyoutItem>
+    <MenuFlyoutItem>Sort by album name</MenuFlyoutItem>
+    <MenuFlyoutItem>Sort by genre</MenuFlyoutItem>
+    <MenuFlyoutSubItem Text="Sort by other...">
+        <MenuFlyoutItem>Sort by rating</MenuFlyoutItem>
+        <MenuFlyoutItem>Sort by most played</MenuFlyoutItem>
+        <MenuFlyoutItem>Sort by most recently listened to</MenuFlyoutItem>
+    </MenuFlyoutSubItem>
+</MenuFlyout>
+```
+
+The `MenuFlyoutSubItem` class represents a "parent" MenuFlyout item. It has an
+[Items](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.menuflyout.items?view=winrt-19041) 
+property, where you can assign all of its child items.  Here's the above example with labels showing which item is which type:
+
+![Anatomy of a MenuFlyout](images/anatomy-diagram.PNG)
+
+You also have the option of using 
+[RadioMenuFlyoutItem](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.radiomenuflyoutitem?view=winui-2.5), 
+which has a [GroupName](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.radiomenuflyoutitem.groupname?view=winui-2.5#Microsoft_UI_Xaml_Controls_RadioMenuFlyoutItem_GroupName) 
+property. Items with the same GroupName will be a part of the same selection model - only one of the items in a Group can be "selected" and display the selection indicator at a time:
+
+```xaml
+<AppBarButton>View
+    <AppBarButton.Flyout>
+        <MenuFlyout>
+            <MenuFlyoutItem>Open</MenuFlyoutItem>
+            <MenuFlyoutSeparator/>
+            <RadioMenuFlyoutItem>Landscape</RadioMenuFlyoutItem>
+            <RadioMenuFlyoutItem IsChecked="True">Portrait</RadioMenuFlyoutItem>
+            <MenuFlyoutSeparator/>
+            <RadioMenuFlyoutItem>Small icons</RadioMenuFlyoutItem>
+            <RadioMenuFlyoutItem IsChecked="True">Medium icons</RadioMenuFlyoutItem>
+            <RadioMenuFlyoutItem>Large icons</RadioMenuFlyoutItem>
```

Comment:
Missing `GroupName` attributes on these items. Kind of important since that's the purpose of the example.

```suggestion
            <RadioMenuFlyoutItem GroupName="Orientation">Landscape</RadioMenuFlyoutItem>
            <RadioMenuFlyoutItem GroupName="Orientation" IsChecked="True">Portrait</RadioMenuFlyoutItem>
            <MenuFlyoutSeparator/>
            <RadioMenuFlyoutItem GroupName="IconSize">Small icons</RadioMenuFlyoutItem>
            <RadioMenuFlyoutItem GroupName="IconSize" IsChecked="True">Medium icons</RadioMenuFlyoutItem>
            <RadioMenuFlyoutItem GroupName="IconSize">Large icons</RadioMenuFlyoutItem>
```